weiqsun_rmd_a2.md

```diff
@@ -0,0 +1,8 @@
+#Ann Arbor, MI
+
+Where UofM is located
+
+### reat place for fun and entertainment
```

Comment:
We use to eat at the Old German, maybe it should make the list if it is still open.